File: vertpro/src/main/java/com/ping/test/server/EchoServer.java
package com.ping.test.server;

import io.vertx.core.AbstractVerticle;
/**
 * Vertx-Java命令行运行
 * vertx run EchoServer.java 
 * @author zhangfei
 *
 */
public class EchoServer extends AbstractVerticle {
	public void start() {
		vertx.createHttpServer().requestHandler(req -> {
			req.response().putHeader("content-type", "text/plain").end("Hello from Vert.x!");
		}).listen(8080);
	}
}

File: vertpro/src/main/java/com/ping/test/server/echo_server.js
/**
 * vertx run echo_server.js
 */
//echo_server.js
vertx.createHttpServer().requestHandler(function (req) {
    req.response().putHeader("content-type", "text/html").end("<html><body><h1>
    Hello from vert.x!</h1></body></html>");
}).listen(8080); 